### ISSUE TITLE
SPIKE: Identify gaps and certify WCAG 2.0 compliance

### ISSUE THREAD

**Issue by dafeder** [GetDKAN/dkan#2236]
The WCAG 2.0 standard has become a universal one for accessibility, and is currently more important than (and mostly supercedes by law) the US-centric Section 508. We should generally start looking toward this standard in all of our accessibility language for DKAN, but also:

- [ ] Specifically identify the ways in which out-of-the-box DKAN does not meet level AA compliance for WCAG 2.0
- [ ] Provide a transparent process for determining individual sites' compliance with level AA
- [ ] (Maybe) provide the option to include a compliance icon in site footer: 

<a href="http://www.w3.org/WAI/WCAG2AA-Conformance"
      title="Explanation of WCAG 2.0 Level Double-A Conformance">

  <img height="32" width="88"
          src="http://www.w3.org/WAI/wcag2AA"
          alt="Level Double-A conformance,
          W3C WAI Web Content Accessibility Guidelines 2.0"></a>

**Comment by janette** [comment]
### Items to fix
- Main navigation: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 7:1, but text in this element has a contrast ratio of 4.79:1. Recommendation: change background to #005c98.
- hero region: 
  * The heading structure is not logically nested. This h2 element appears to be the primary document heading, so should be an h1 element.
  * This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 7:1, but text in this element has a contrast ratio of 6.85:1. Recommendation: change background to #005c97.
- Featured Topics: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 7:1, but text in this element has a contrast ratio of 4.79:1. Recommendation: change text colour to #005c98.
- footer link color: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 7:1, but text in this element has a contrast ratio of 4.79:1. Recommendation: change text colour to #005c98.
- Search page:
   * Results count: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 3:1, but text in this element has a contrast ratio of 1.66:1. Recommendation: change text colour to #949494.
  * Resource 'links' data-format="csv" : This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.01:1. Recommendation: change background to #917300.
  * group/org link: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.85:1. Recommendation: change text colour to #767676.
- Dataset page:
  * Additional info table: The relationship between td elements and their associated th elements is not defined. Use either the scope attribute on th elements, or the headers attribute on td elements.
- Resource page:
  * This link points to a named anchor "remove" within the document, but no anchor exists with that name.
  * data preview add filter link: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.39:1. Recommendation: change background to #f8f8f8.
  * data preview remove filter 'x': This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.39:1. Recommendation: change background to #f8f8f8.
  * data preview add filter text input: This textinput element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined. This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.

**Comment by dharizza** [comment]
I updated everything related to contrast to comply with AAA level. 
Also, fixed heading structure in homepage.
I had some issues with the anchors in the resource page and also with the table in the datasets page.
PR related: https://github.com/GetDKAN/dkan/pull/2921
